lisa/features/disks.py

```diff
@@ -74,10 +74,11 @@ def get_resource_disk_mount_point(self) -> str:
         raise NotImplementedError
 
     def get_resource_disks(self) -> List[str]:
-        return []
+        raise NotImplementedError
```

Comment:
Why change this? The resource disk is cloud specific feature, so it should return empty by default.